pkgs/development/python-modules/yarg/default.nix

```diff
@@ -3,36 +3,41 @@
   buildPythonPackage,
   fetchFromGitHub,
   requests,
-  nose,
   mock,
+  setuptools,
+  pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "yarg";
   version = "0.1.9";
-  format = "setuptools";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "kura";
-    repo = pname;
+    repo = "yarg";
     rev = version;
-    sha256 = "1isq02s404fp9whkm8w2kvb2ik1sz0r258iby0q532zw81lga0d0";
+    hash = "sha256-oAH1aED8i1Ew8CuiIjL4Oswo1p6CozohT9cRQLQAWMc=";
   };
 
-  propagatedBuildInputs = [ requests ];
+  patchPhase = ''
+    substituteInPlace tests/test_package.py tests/test_release.py \
+      --replace-fail "assertEquals" "assertEqual"
+  '';
```

Comment:
I know this package has been mostly dormant for years, but it looks like there was a PR merged as recently as two years ago – do you think you’d be up for sending this patch and one to remove the nose dependencies upstream so the rest of the ecosystem can benefit? Still better to have this than not if that seems like too much fuss for a package that’s basically unmaintained anyway though!

Reply:
I'll make one and submit a PR, and just add the commit patch here. If it's merged and a release is made, great! If not, the commit still exists, and the patch will work just fine.

Reply:
I've updated the package. It's now the latest commit of yarg(needed because it improves a few things and also lets my patch actually apply cleanly), and my patch is added, which allows tests to pass.